packages/core/src/auth/providers/credentialsProviderManager.ts

```diff
@@ -52,7 +52,7 @@ export class CredentialsProviderManager {
                 continue
             }
 
-            void this.emitWithDebounce({
+            telemetry.aws_loadCredentials.emit({
```

Comment:
Does removing the debounce here mean we think this piece is no longer the issue?

Reply:
yep, and now behaves slightly differently.